build/templates/enums.py.mako

```diff
@@ -15,7 +15,7 @@ class ${enum_name}(Enum):
     % else:
     ${enum_value['name']} = ${enum_value['value']}
     % endif
-    % if 'documentation' in enum_value:
+    % if 'documentation' in enum_value and len(helper.get_documentation_for_node_docstring(enum_value, config, indent=4).strip()) > 0:
```

Comment:
@texasaggie97 What do you think of adding the processed documentation to the metadata instead?

Reply:
Which processing would we do? All of them? docstring + rst even though at most we'd only need one?

Reply:
I was thinking all of them.

> I don't think that would be appropriate for this change though.

I agree.

For this change:
"if 'documentation' in enum_value" I think was very clear.
The new thing "and len(helper.get_documentation_for_node_docstring(enum_value, config, indent=4).strip()) > 0" not so much.

What can we do here for added clarity? What is this doing?